float32 mask_x_left

float32 mask_x_right

float32 mask_y_top

float32 mask_y_bot

float32 center_target

float32 vel_x

float32 vel_y_P

float32 vel_y_D

float32 vel_z_P

float32 vel_z_D

int8 en
